core-bundle/contao/classes/Backend.php

```diff
@@ -544,6 +544,11 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 				}
 			}
 
+			if (($GLOBALS['TL_DCA'][$strTable]['list']['sorting']['useNewDesign'] ?? false) && \in_array(Input::get('act'), array('paste', 'select', null), true))
```

Comment:
Do you think that's necessary? How many "new designs" will we get? I mean, why is it configurable? We don't have any BC on how the back end looks like, templates, styles etc.

Reply:
Of course it is necessary, otherwise it wouldn‘t be there. 😄 

However, it is not there for user configuration. The new design does not make sense for all parent views, so I needed a way to specify where it shall be applied. We can change the name of the config option if you prefer.

Reply:
This is how the modules view would look like with the new design:

<img width="1186" alt="" src="https://user-images.githubusercontent.com/1192057/209852881-25debadc-8ed1-47c4-b400-175537e20fac.png">

That‘s why we need the config option.

Reply:
Ah I see. That was actually the question ;) I'd prefer a different name, yes. Because what's after "new design"? "new design 2", "new design 3"? 😊 I don't know what name comes to your mind? "Boxed layout"? Anyway, would also be good to read in the CHANGELOG as such: "Introduced new xy layout". Wdyt?

Reply:
I have renamed the option to `renderAsGrid` in e609c6d20d4624c733653e6fc1d5affa9716414f because that is what we actually need to distinguish (render as table vs render as grid). To remain backwards compatible, "render as table" is the default.

Reply:
Very good, I like that :)